extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_linkAuthentication.php

```diff
@@ -1,6 +1,6 @@
 <?php if ( isset( $model['subtitle'] ) ): ?>
 	<div class="wds-global-navigation__account-menu-dropdown-caption"><?= DesignSystemHelper::renderText( $model['subtitle'] ) ?></div>
 <?php endif; ?>
-<button class="wds-button wds-is-full-width <?= isset( $model['subtitle'] ) ? 'wds-is-secondary' : ''; ?>">
+<a href="<?= Sanitizer::encodeAttribute( $model['href'] ); ?>" id="<?= Sanitizer::encodeAttribute( $model['title']['key'] ); ?>" class="wds-button wds-is-full-width <?= isset($model['subtitle'] ) ? 'wds-is-secondary' : ''; ?>">
```

Comment:
using `subtitle` for defining class names doesn't seem to be right approach here.